src/git.ts

```diff
@@ -216,21 +209,19 @@ export async function getViaGit(
               LC_CTYPE: 'C.UTF-8',
               CHERE_INVOKING: '1',
               MSYSTEM: 'MINGW64',
-              PATH: `${gitForWindowsUsrBinPath}${delimiter}${process.env.PATH}`
+              PATH: `${gitCmdPath}${delimiter}${gitForWindowsUsrBinPath}${delimiter}${process.env.PATH}`
```

Comment:
I added `/cmd` here, but I think your [suggestion](https://github.com/git-for-windows/setup-git-for-windows-sdk/pull/952) is even better.

Do you think the rest of this PR would still add some value (extracting the wrapper and adding tests)?

Reply:
It's a lot of code... I'm unsure. At this point, I want to focus on getting it to work, and I don't think I'll have the energy for this type of cleanup afterwards 😃